rml/.htaccess

```diff
@@ -0,0 +1,106 @@
+RewriteRule rml/GraphMap$ https://kg-construct.github.io/rml-core-spec/spec/docs#http://w3id.org/rml/GraphMap
+RewriteRule rml/GraphMap/$ https://kg-construct.github.io/rml-core-spec/spec/docs#http://w3id.org/rml/GraphMap
+RewriteRule rml/DatatypeMap$ https://kg-construct.github.io/rml-core-spec/spec/docs#http://w3id.org/rml/DatatypeMap
+RewriteRule rml/DatatypeMap/$ https://kg-construct.github.io/rml-core-spec/spec/docs#http://w3id.org/rml/DatatypeMap
+
+# Ontology documentation
+RewriteRule rml/core$ https://kg-construct.github.io/rml-core-spec/ontology/documentation/index-en.html
+RewriteRule rml/core/$ https://kg-construct.github.io/rml-core-spec/ontology/documentation/index-en.html
+
+# === RML Source and Target ===
+# Specification
+RewriteRule rml/io/spec$ https://kg-construct.github.io/rml-io-spec/spec/docs
+RewriteRule rml/io/spec/$ https://kg-construct.github.io/rml-io-spec/spec/docs
+
+# Resources: LogicalSource, LogicalTarget, Source, Target
+RewriteRule rml/LogicalSource$ https://kg-construct.github.io/rml-io-spec/spec/docs#http://w3id.org/rml/LogicalSource
+RewriteRule rml/LogicalSource/$ https://kg-construct.github.io/rml-io-spec/spec/docs#http://w3id.org/rml/LogicalSource
+RewriteRule rml/LogicalTarget$ https://kg-construct.github.io/rml-io-spec/spec/docs#http://w3id.org/rml/LogicalTarget
+RewriteRule rml/LogicalTarget/$ https://kg-construct.github.io/rml-io-spec/spec/docs#http://w3id.org/rml/LogicalTarget
+RewriteRule rml/Source$ https://kg-construct.github.io/rml-io-spec/spec/docs#http://w3id.org/rml/Source
+RewriteRule rml/Source/$ https://kg-construct.github.io/rml-io-spec/spec/docs#http://w3id.org/rml/Source
+RewriteRule rml/Target$ https://kg-construct.github.io/rml-io-spec/spec/docs#http://w3id.org/rml/Target
+RewriteRule rml/Target/$ https://kg-construct.github.io/rml-io-spec/spec/docs#http://w3id.org/rml/Target
+
+# Ontology documentation
+RewriteRule rml/io$ https://kg-construct.github.io/rml-io-spec/ontology/documentation/index-en.html
+RewriteRule rml/io/$ https://kg-construct.github.io/rml-io-spec/ontology/documentation/index-en.html
+
+# === RML Collections and Containers ===
+# Specification
+RewriteRule rml/cc/spec$ https://kg-construct.github.io/rml-cc-spec/spec/docs
+RewriteRule rml/cc/spec/$ https://kg-construct.github.io/rml-cc-spec/spec/docs
+
+# Resources: GatherMap
+RewriteRule rml/GatherMap$ https://kg-construct.github.io/rml-cc-spec/spec/docs#http://w3id.org/rml/GatherMap
+RewriteRule rml/GatherMap/$ https://kg-construct.github.io/rml-cc-spec/spec/docs#http://w3id.org/rml/GatherMap
+
+# Ontology documentation
+RewriteRule rml/cc$ https://kg-construct.github.io/rml-cc-spec/ontology/documentation/index-en.html
+RewriteRule rml/cc/$ https://kg-construct.github.io/rml-cc-spec/ontology/documentation/index-en.html
+
+# === RML FNML ===
+# Specification
+RewriteRule rml/fnml/spec$ https://kg-construct.github.io/rml-fnml-spec/spec/docs
```

Comment:
the spec is online at https://kg-construct.github.io/fnml-spec/

Reply:
After the meeting today, the Redirections spreadsheet is finalized. Only the FNML spec needs to update its repo now to adjust its URL correctly for Github Pages.